awx/main/tasks/host_metrics.py

```diff
@@ -0,0 +1,202 @@
+import datetime
+from dateutil.relativedelta import relativedelta
+import logging
+
+from django.conf import settings
+from django.db.models import Count
+from django.db.models.functions import TruncMonth
+from django.utils.timezone import now
+from awx.main.dispatch import get_task_queuename
+from awx.main.dispatch.publish import task
+from awx.main.models.inventory import HostMetric, HostMetricSummaryMonthly
+from awx.conf.license import get_license
+
+logger = logging.getLogger('awx.main.tasks.host_metric_summary_monthly')
+
+
+@task(queue=get_task_queuename)
+def host_metric_summary_monthly():
+    """Run cleanup host metrics summary monthly task each week"""
+    from awx.conf.models import Setting
+
+    if _is_run_threshold_reached(
+            Setting.objects.filter(key='HOST_METRIC_SUMMARY_TASK_LAST_TS').first(),
+            getattr(settings, 'HOST_METRIC_SUMMARY_TASK_INTERVAL', 7) * 86400
+    ):
+        logger.info("TASK STATUS: host_metric_summary_monthly: STARTED")
```

Comment:
Might be confusing log, since _other_ logs will say that it's starting the `host_metric_summary_monthly` task when the time criteria is not met. I'd suggest slight modification like

```python
logger.info(f"Executing host_metric_summary_monthly, last ran at {settings.HOST_METRIC_SUMMARY_TASK_LAST_TS}")
```

Otherwise the level and everything like that seems appropriate :+1: